main/control/user.py

```diff
@@ -24,7 +24,8 @@
 @app.route('/admin/user/')
 @auth.admin_required
 def user_list():
-  user_dbs, user_cursor = model.User.get_dbs(email=util.param('email'))
+  user_dbs, user_cursor, prev_cursor =\
+      model.User.get_dbs(email=util.param('email'), prev_cursor=True)
```

Comment:
also maybe instead of `user_cursor`, `prev_cursor` something like `user_next`, `user_prev`.. WDYT? (`user_next_cursor` is too big :rabbit: )

The reason that I switched to `user_cursor` instead of just `next_cursor` was because of some conflicts if there were more queries within one function...